test/onevariable.jl

```diff
@@ -449,7 +456,7 @@ end
         A_mul_B!(Y,A,B)
 
         # do we get the same result when using the `A*B` form?
-        @test A*B==Y
+        @test A*B≈Y
```

Comment:
Thanks @blas-ko for this change! I'd only insist in specifying both `atol` and `rtol` as:

`@test A*B ≈ Y atol=0.0 rtol=eps()`

otherwise, since we are being consistent with `Base.isapprox`, the default `rtol` value is `sqrt(eps())`, which is ~1e-8